packages/appium/test/e2e/driver.e2e.spec.js

```diff
@@ -536,6 +536,61 @@ describe('FakeDriver via HTTP', function () {
       }
     });
   });
+
+  describe('Bidi protocol', function () {
```

Comment:
would it be possible to add at least one test where we check BiDi stuff with https enabled? Check https://github.com/appium/appium/blob/d972a6a0eb3dbee0468700426be668807e20ca0c/packages/base-driver/test/e2e/express/server.e2e.spec.js#L110 for examples

Reply:
sure i will try that

Reply:
ok i added a test but it looks like wdio has a bug in that it will not allow self signed certs for the bidi connection. hoping @christian-bromann will fix that soon :trollface:

Reply:
have you tried [this](https://stackoverflow.com/questions/10888610/ignore-invalid-self-signed-ssl-certificate-in-node-js-with-https-request) workaround?

Reply:
i submitted a PR and it has been released. at long last @mykola-mokhnach CI is green!